### ISSUE TITLE
"Waiting dots" bug  

### ISSUE THREAD

**Issue by panr** [rsms/fb-mac-messenger#316]
<img width="121" alt="waiting-dots-error" src="https://cloud.githubusercontent.com/assets/1303365/17883046/890fae70-6910-11e6-9e0d-89f57edf7801.png">

There's a bug with those animated dots. The last one renders itself below to row, like this ➝ `⠍`

Version: Messenger 0.1.9

**Comment by Gunny123** [comment]
@panr @rsms 

This bug is still hanging out in version 0.1.10 as well. 

![bug screenshot](https://cloud.githubusercontent.com/assets/10899790/21034171/99d3b9d2-bd83-11e6-84a4-63af57871ba8.png)

Bugs my OCD.

**Comment by jamesinc** [comment]
Bug still present in 0.1.11.

**Comment by thatchrismiller** [comment]
I'm still seeing this too.

**Comment by Gunny123** [comment]
@thatchrismiller I don't think this is going to be fixed for a loooong time. Especially now that [Facebook is piloting a dedicated desktop application](https://9to5mac.com/2017/10/05/facebook-macos-app-workplace/).

**Comment by panr** [comment]
@rsms any updates? 😏

![waiting](https://user-images.githubusercontent.com/1303365/44396689-fbbf3380-a53d-11e8-894f-2583ac3c238c.png)
